doc/main/hubs/cityhub.rst

```diff
@@ -19,6 +19,17 @@ City Hub
 
     .. automethod:: pybricks.hubs::CityHub.light.animate
 
+    .. rubric:: Using connectionless Bluetooth messaging
+
+    .. note:: ``ble.broadcast()`` does not work on ``CityHub`` due to a
```

Comment:
Can be regular text; does not need to grab the user attention while they are scrolling to read about lights/buttons/etc.